TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift

```diff
@@ -35,6 +35,20 @@ enum KeyAttrClass {
   }
 }
 
+public enum KeyAttrAccessible {
+  case afterFirstUnlock
```

Comment:
Please add documentation to each value, to describe behavior and what to expect

Reply:
Also, I think this enum should be closer to the `TwilioVerify` class, adding it here seems hidden

Reply:
Make sense, Added documentation & moved to `TwilioVerify` class, Thank you 👍